src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/PdfGeneration/HealthCertificatePDFVersionViewModel.swift

```diff
@@ -0,0 +1,33 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import PDFKit
+import OpenCombine
+
+class HealthCertificatePDFVersionViewModel {
+
+	// MARK: - Init
+
+	init(
+		healthCertificate: HealthCertificate,
+		pdfView: PDFView
+	) {
+		self.healthCertificate = healthCertificate
+		self.pdfView = pdfView
+	}
+
+	// MARK: - Internal
+
+	let healthCertificate: HealthCertificate
+	let pdfView: PDFView
```

Comment:
A ViewModel holder the view looks strange to me

Reply:
Of the architectural view, you are right. I will change it so that we receive the pdfDocument and hold this.